### PR TITLE
Action Engine grpc configuration.

### PR DESCRIPTION
**Pull request by mwwolters**
Thanks for opening a Pull Request!

Here's a handy checklist to ensure your PR goes smoothly.

- [x] I signed Google's [Contributor License Agreement](https://opensource.google.com/docs/cla/)
- [x] My code conforms to Google's [python style](https://google.github.io/styleguide/pyguide.html).
- [x] My PR at a minimum doesn't decrease unit-test coverage (if applicable).
- [x] My PR has been functionally tested.
- [x] All of the [unit-tests](http://forsetisecurity.org/docs/development/#executing-tests) still pass.
- [x] Running `pylint --rcfile=pylintrc` passes.

These guidelines and more can be found in our [contributing guidelines](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: google/cloud/forseti/services/action_engine.proto [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+message ListResponse {
+  repeated Action actions = 1;
+}
+
+message RuleResult {
+  // The id of the rule, must be unique
+  string rule_id = 1;
+  // The result of the rule.
+  bool result = 2;
```
Comment (ahoying) [inline google/cloud/forseti/services/action_engine.proto]: What does this bool represent? What does true mean, or false?

File: google/cloud/forseti/services/action_engine.proto [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  google.protobuf.Timestamp timestamp = 6;
+  // A human readable string output of what this result says.
+  string info = 7;
+}
+
+message ProcessResultsRequest {
+  repeated RuleResult results = 1;  // TODO define rule results
+}
+
+message Result {
```
Comment (ahoying) [inline google/cloud/forseti/services/action_engine.proto]: I recommend renaming to ActionResult.

File: google/cloud/forseti/services/action_engine.proto [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  enum Code {
+    SUCCESS = 0;
+    ERROR = 1;
+    NOT_PROCESSED = 2;  // Used if no actions processed the result.
+  }
+  Code code = 1;
+  string info = 2;
```
Comment (ahoying) [inline google/cloud/forseti/services/action_engine.proto]: Is there any other state information that the Action engine should return along with a result?

File: google/cloud/forseti/services/action_engine.proto (hunk 1 above)
Comment (mwwolters) [reply google/cloud/forseti/services/action_engine.proto]: Removed based on discussion.

File: google/cloud/forseti/services/action_engine.proto (hunk 2 above)
Comment (mwwolters) [reply google/cloud/forseti/services/action_engine.proto]: Done.

File: google/cloud/forseti/services/action_engine.proto (hunk 3 above)
Comment (mwwolters) [reply google/cloud/forseti/services/action_engine.proto]: Added action id.

File: google/cloud/forseti/services/action_engine.proto [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+}
+
+message ProcessedResult {
+  repeated RuleResult result = 1;
```
[the remark this answers is not in the document]
Comment (mwwolters) [reply google/cloud/forseti/services/action_engine.proto]: Done.

File: google/cloud/forseti/services/action_engine.proto [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+    NOT_PROCESSED = 2;  // Used if no actions processed the result.
+  }
+  Code code = 1;
+  string action_id = 2;
+  string info = 3;
+}
+
+message ProcessedResult {
+  RuleResult result = 1;
+  repeated ActionResult action_result = 2;
```
Comment (ahoying) [inline google/cloud/forseti/services/action_engine.proto]: Nit, action_results.

**Pull request merged**